entrypoint.sh

```diff
@@ -5,14 +5,17 @@ set -e
 
 APT_DEPENDENCIES=$1
 CODECOV_TOKEN=$2
-SCRIPT_BEFORE_CMAKE=$3
-CMAKE_ARGS=$4
-SCRIPT_BETWEEN_CMAKE_MAKE=$5
-SCRIPT_AFTER_MAKE=$6
-SCRIPT_AFTER_MAKE_TEST=$7
+CMAKE_ARGS=$3
+
+SOURCE_DEPENDENCIES=".github/ci-bionic/dependencies.yaml"
```

Comment:
I noticed these weren't consistent in the other PR, you could fix it up here: https://github.com/ignition-tooling/ubuntu-bionic-ci-action/pull/13#discussion_r457702567

Reply:
Here you go: a2fd4452f23e664997f1d85d0309147acf5a4527